satellite/repair/repairer/segments.go

```diff
@@ -117,6 +117,17 @@ func (repairer *SegmentRepairer) Repair(ctx context.Context, path storj.Path) (s
 	if pointer.Remote.Redundancy.Total != 0 {
 		healthyRatioBeforeRepair = float64(numHealthy) / float64(pointer.Remote.Redundancy.Total)
 	}
+	mon.Meter("repair_attempts").Mark(1)
```

Comment:
these stats are sent earlier in this function. If you are consolidating them, make sure to remove the old calls

Reply:
this wasn't supposed to be comitted :/ i was just collecting the stuff for pasting into slack